net/aircrack-ng/Makefile

```diff
@@ -8,27 +8,41 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=aircrack-ng
-PKG_VERSION:=1.2-rc1
-PKG_RELEASE:=3
+PKG_VERSION:=1.4
+PKG_RELEASE:=1
 PKG_LICENSE:=GPL-2.0
 PKG_LICENSE_FILES:=LICENSE
 
 PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
-PKG_SOURCE_URL:=http://download.aircrack-ng.org/ \
-		http://archive.aircrack-ng.org/aircrack-ng/$(PKG_VERSION)/
-PKG_HASH:=cf3134521e1c3d7aed4e384e3e5e7b6959e2d485bd1554474608a3a9328e35fd
+PKG_SOURCE_URL:=https://github.com/aircrack-ng/aircrack-ng/archive/$(PKG_VERSION)/
+PKG_HASH:=425122525e9c7dd8834d75b6716516f2b673342b7e04fa247ec277a8c8433b47
 
+PKG_BUILD_DIR:=$(BUILD_DIR)/aircrack-ng-$(PKG_VERSION)
 PKG_BUILD_PARALLEL:=1
 PKG_INSTALL:=1
+PKG_FORTIFY_SOURCE:=0
 
 PKG_MAINTAINER:=Rick Farina <zerochaos@gentoo.org>
 
+PKG_FIXUP:=autoreconf
```

Comment:
With the official tarball, this is probably not needed. Unless there's some upstream issue.

Reply:
I'm the upstream. :)

Yeah, unfortunately - it's needed for odd reasons...